<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.4.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Tim Kacprowski">

  
  
  
    
  
  <meta name="description" content="Resarch Group Computational Systems Medicine">

  
  <link rel="alternate" hreflang="en-us" href="https://compsysmed.de/publication/">

  


  

  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.863679108eccaa86b806f7003d41cf89.css">

  

  
  
  

  
  <link rel="alternate" href="/publication/index.xml" type="application/rss+xml" title="ResGro CompSysMed">
  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon-32.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://compsysmed.de/publication/">

  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="ResGro CompSysMed">
  <meta property="og:url" content="https://compsysmed.de/publication/">
  <meta property="og:title" content="Publications | ResGro CompSysMed">
  <meta property="og:description" content="Resarch Group Computational Systems Medicine"><meta property="og:image" content="https://compsysmed.de/img/icon-192.png">
  <meta property="twitter:image" content="https://compsysmed.de/img/icon-192.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2020-12-26T00:00:00&#43;01:00">
  

  

<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css">
<script src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
<script>
  window.addEventListener("load", function(){
    window.cookieconsent.initialise({
      "palette": {
        "popup": {
          "background": "#2962ff",
          "text": "#fff"
        },
        "button": {
          "background": "#fff",
          "text": "#2962ff"
        }
      },
      "theme": "classic",
      "content": {
        "message": "This website uses cookies to ensure you get the best experience on our website.",
        "dismiss": "Got it!",
        "link": "Learn more",
        "href": "https://cookies.insites.com"
      }
    })});
</script>



  





  <title>Publications | ResGro CompSysMed</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">ResGro CompSysMed</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span><b>News</b></span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#people"><span><b>Team</b></span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects"><span><b>Projects</b></span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#softress"><span><b>Software & Resources</b></span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span><b>Publications</b></span></a>
        </li>

        
        

      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a href="https://www.tum.de" target="_blank">
            <img src="/img/TUM_Web_Logo_blau.png" alt="TUM" style="margin-top:12px;margin-left:12px;">
          </a>
        </li>

      </ul>

    </div>
  </div>
</nav>


  












  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1 itemprop="name">Publications</h1>

  

  
</div>



<div class="universal-wrapper">
  <div class="row">
    <div class="col-lg-12">

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <div class="form-row mb-4">
        <div class="col-auto">
          <input type="search" class="filter-search" placeholder="Search..." autocapitalize="off"
          autocomplete="off" autocorrect="off" role="textbox" spellcheck="false">
        </div>
        <div class="col-auto">
          <select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group="pubtype">
            <option value="*">Type</option>
            
            
            <option value=".pubtype-2">
              Journal article
            </option>
            
            <option value=".pubtype-3">
              Preprint
            </option>
            
            <option value=".pubtype-6">
              Book section
            </option>
            
          </select>
        </div>
        <div class="col-auto">
          <select class="pub-filters form-control form-control-sm" data-filter-group="year">
            <option value="*">Date</option>
            
            
            
            <option value=".year-2020">
              2020
            </option>
            
            <option value=".year-2019">
              2019
            </option>
            
            <option value=".year-2018">
              2018
            </option>
            
            <option value=".year-2017">
              2017
            </option>
            
            <option value=".year-2016">
              2016
            </option>
            
            <option value=".year-2015">
              2015
            </option>
            
            <option value=".year-2014">
              2014
            </option>
            
            <option value=".year-2013">
              2013
            </option>
            
            <option value=".year-2012">
              2012
            </option>
            
            <option value=".year-2011">
              2011
            </option>
            
            
          </select>
        </div>
      </div>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-12-26 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2020-12-26 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-lazareva-s.-canzar-k.-yuan-j.-baumbach-d.-blumenthal-m.-list-t.-kacprowski/">O. Lazareva, S. Canzar, K. Yuan, J. Baumbach, D. Blumenthal, M. List*, <strong>T. Kacprowski</strong>*</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>December 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/lazareva-et-al.-2020.3/" itemprop="url">BiCoN: Network-constrained biclustering of patients and omics data</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Unsupervised learning approaches are frequently employed to stratify patients into clinically relevant subgroups and to identify biomarkers such as disease-associated genes. However, clustering and biclustering techniques are oblivious to the functional relationship of genes and are thus not ideally suited to pinpoint molecular mechanisms along with patient subgroups.<br> We developed the network-constrained biclustering approach BiCoN (Biclustering Constrained by Networks) which (i) restricts biclusters to functionally related genes connected in molecular interaction networks and (ii) maximizes the difference in gene expression between two subgroups of patients. This allows BiCoN to simultaneously pinpoint molecular mechanisms responsible for the patient grouping. Network-constrained clustering of genes makes BiCoN more robust to noise and batch effects than typical clustering and biclustering methods. BiCoN can faithfully reproduce known disease subtypes as well as novel, clinically relevant patient subgroups, as we could demonstrate using breast and lung cancer datasets. In summary, BiCoN is a novel systems medicine tool that combines several heuristic optimization strategies for robust disease mechanism extraction. BiCoN is well-documented and freely available as a python package or a web interface.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa1076" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-12-21 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2020-12-21 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-wu-t.-kacprowski-d.-zehn/">M. Wu, <strong>T. Kacprowski</strong>, D. Zehn</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>December 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/wu-et-al.-2020/" itemprop="url">PySCNet: A tool for reconstructing and analyzing gene regulatory network from single-cell RNA-Seq data</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The Advanced capacities of high throughput single cell technologies have facilitated a great understanding of complex biological systems, ranging from cell heterogeneity to molecular expression kinetics. Several pipelines have been introduced to standardize the scRNA-seq analysis workflow. These include cell population identification, cell marker detection and cell trajectory reconstruction. Yet, establishing a systematized pipeline to capture regulatory relationships among transcription factors (TFs) and genes at the cellular level still remains challenging. Here we present PySCNet, a python toolkit that enables reconstructing and analyzing gene regulatory networks (GRNs) from single cell transcriptomic data. PySCNet integrates competitive gene regulatory construction methodologies for cell specific or trajectory specific GRNs and allows for gene co-expression module detection and gene importance evaluation. Moreover, PySCNet offers a user-friendly dashboard website, where GRNs can be customized in an intuitive way.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.12.18.423482" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-09-25 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-09-25 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/z.-louadi/"><strong>Z. Louadi</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/k.-yuan/">K. Yuan</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-gress/">A. Gress</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-tsoy/">O. Tsoy</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-v.-kalinina/">O. V. Kalinina</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-baumbach/">J. Baumbach</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-list/">M. List</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nucleic Acid Research</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/louadi-et-al.-2020/" itemprop="url">DIGGER: exploring the functional role of alternative splicing in protein interactions</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Alternative splicing plays a major role in regulating the functional repertoire of the proteome. However, isoform-specific effects to protein-protein interactions (PPIs) are usually overlooked, making it impossible to judge the functional role of individual exons on a systems biology level. We overcome this barrier by integrating protein-protein interactions, domain-domain interactions and residue-level interactions information to lift exon expression analysis to a network level. Our user-friendly database DIGGER is available at <a href="https://exbio.wzw.tum.de/digger" target="_blank">https://exbio.wzw.tum.de/digger</a> and allows users to seamlessly switch between isoform and exon-centric views of the interactome and to extract sub-networks of relevant isoforms, making it an essential resource for studying mechanistic consequences of alternative splicing.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/nar/gkaa768" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-09-10 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-09-10 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/y.-bouter/">Y. Bouter</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><strong>F. Rößler</strong></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-r.-jensen/">L. R. Jensen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-w.-kuss/">A. W. Kuss</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-a.-bayer/">T. A. Bayer</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Frontiers in Neuroscience</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/bouter-et-al.-2020/" itemprop="url">MiRNA Alterations Elicit Pathways Involved in Memory Decline and Synaptic Function in the Hippocampus of Aged Tg4-42 Mice</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The transcriptome of non-coding RNA (ncRNA) species is increasingly focused in Alzheimer’s disease (AD) research. NcRNAs comprise, among others, transfer RNAs, long non-coding RNAs and microRNAs (miRs), each with their own specific biological function. We used smallRNASeq to assess miR expression in the hippocampus of young (3 month old) and aged (eight month old) Tg4-42 mice, a model system for sporadic AD, as well as age-matched wildtype controls. Tg4-42 mice express N-truncated Aβ4-42, develop age-related neuron loss, reduced neurogenesis and behavioral deficits. Our results do not only confirm known miR-AD associations in Tg4-42 mice, but more importantly pinpoint 22 additional miRs associated to the disease. Twenty-five miRs were differentially expressed in both aged Tg4-42 and aged wildtype mice while eight miRs were differentially expressed only in aged wildtype mice, and 33 only in aged Tg4-42 mice. No significant alteration in the miRNome was detected in young mice, which indicates that the changes observed in aged mice are down-stream effects of A -induced pathology in the Tg4-42 mouse model for AD. Targets of those miRs were predicted using miRWalk. For miRs that were differentially expressed only in the Tg4-42 model, 128 targets could be identified, whereas 18 genes were targeted by miRs only differentially expressed in wildtype mice and 85 genes were targeted by miRs differentially expressed in both mouse models. Genes targeted by differentially expressed miRs in the Tg4-42 model were enriched for negative regulation of long-term synaptic potentiation, learning or memory, regulation of trans-synaptic signaling and modulation of chemical synaptic transmission obtained. This untargeted miR sequencing approach supports previous reports on the Tg4-42 mice as a valuable model for AD. Furthermore, it revealed miRs involved in AD, which can serve as biomarkers or therapeutic targets.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fnins.2020.580524" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-06-05 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-06-05 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person">R. Nasirigerdeh, and R. Torkzadehmahani, J. Matschinske, T. Frisch, M. List, J. Späth, S. Weiß, U. Völker, N. K. Wenke, <strong>T. Kacprowski</strong>, J. Baumbach</span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/nasirigerdeh-et-al.-2020/" itemprop="url">sPLINK: A Federated, Privacy-Preserving Tool as a Robust Alternative to Meta-Analysis in Genome-Wide Association Studies</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Genome-wide association studies (GWAS) have been widely used to unravel connections between genetic variants and diseases. Larger sample sizes in GWAS can lead to discovering more associations and more accurate genetic predictors. However, sharing and combining distributed genomic data to increase the sample size is often challenging or even impossible due to privacy concerns and privacy protection laws such as the GDPR. While meta-analysis has been established an effective approach to combine summary statistics of several GWAS, its accuracy can be attenuated in the presence of cross-study heterogeneity. Here, we present sPLINK (safe PLINK), a user-friendly software tool set which performs federated, privacy-preserving GWAS on distributed datasets, preserving accuracy. sPLINK neither exchanges raw data nor does it rely on summary statistics. Instead, it performs model training in a federated manner, communicating only model parameters between cohorts and a central server. We verify that the federated results from sPLINK are the same as those from aggregated analysis conducted with PLINK. Moreover, we demonstrate that sPLINK is robust against imbalanced phenotype distributions across cohorts while existing meta-analysis tools considerably lose accuracy in such scenarios. Federated and user-friendly analysis with sPLINK, thus, has the potential to replace meta-analysis as the gold standard for collaborative GWAS.Competing Interest StatementThe authors have declared no competing interest.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.06.05.136382" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-6 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-lazareva-m.-lautizi-a.-fenn-m.-list-t.-kacprowski-j.-baumbach/">O. Lazareva, <strong>M. Lautizi</strong>, <strong>A. Fenn</strong>, M. List, <strong>T. Kacprowski</strong>, J. Baumbach</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Elsevier</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/lazareva-et-al.-2020.2/" itemprop="url">Multi-Omics Analysis in a Network Context (in &#34;Reference Module in Biomedical Sciences&#34;)</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Multi-omics data analysis holds great potential for treatment optimization, molecular diagnostics and disease prognosis. To gain mechanistic insights into diseases, these complex data can be integrated with prior knowledge of molecular interactions and functional relationships using network enrichment methods. The vast majority of existing approaches use pre-determined molecular pathways and thus unable to reveal novel molecular pathways that explain the disease phenotype. In contrast, de novo network enrichment approaches identify disease-associated subnetworks offering mechanistic insights into a disease. However, we observe that these tools are often released in prototype status, drastically limiting the adoption of the tool due to possible software bugs, insufficient documentation and non-intuitive choice of hyperparameters. In the following article, we provide a complete introduction to the field of network enrichment, discuss the algorithmic background of the existing de novo and non-de novo approaches, discuss their limitations and biases and mention possible development perspectives. We present a comprehensive overview of 19 state-of-the-art, well documented and continuously maintained de novo network enrichment tools and offer guidelines for choosing the most appropriate method for a particular analysis.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/B978-0-12-801238-3.11647-2" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-04-26 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-04-26 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-sadegh-j.-matschinske-d.-b.-blumenthal-g.-galindez-t.-kacprowski-m.-list-r.-nasirigerdeh-m.-oubounyt-a.-pichlmair-t.-d.-rose-m.-salgado-albarran-j.-spath-a.-stukalov-n.-k.-wenke-k.-yuan-j.-k.-pauling-j.-baumbach/"><strong>S. Sadegh</strong>, J. Matschinske, D. B. Blumenthal, <strong>G. Galindez</strong>, <strong>T. Kacprowski</strong>, M. List, R. Nasirigerdeh, M. Oubounyt, A. Pichlmair, T. D. Rose, M. Salgado-Albarrán, J. Späth, A. Stukalov, N. K. Wenke, K. Yuan, J. K. Pauling, J. Baumbach</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>arXiv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/sadegh-et-al.-2020/" itemprop="url">Exploring the SARS-CoV-2 virus-host-drug interactome for drug repurposing</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Coronavirus Disease-2019 (COVID-19) is an infectious disease caused by the SARS-CoV-2 virus. It was first identified in Wuhan, China, and has since spread causing a global pandemic. Various studies have been performed to understand the molecular mechanisms of viral infection for predicting drug repurposing candidates. However, such information is spread across many publications and it is very time-consuming to access, integrate, explore, and exploit. We developed CoVex, the first interactive online platform for SARS-CoV-2 and SARS-CoV-1 host interactome exploration and drug (target) identification. CoVex integrates 1) experimentally validated virus-human protein interactions, 2) human protein-protein interactions and 3) drug-target interactions. The web interface allows user-friendly visual exploration of the virus-host interactome and implements systems medicine algorithms for network-based prediction of drugs. Thus, CoVex is an important resource, not only to understand the molecular mechanisms involved in SARS-CoV-2 and SARS-CoV-1 pathogenicity, but also in clinical research for the identification and prioritization of candidate therapeutics. We apply CoVex to investigate recent hypotheses on a systems biology level and to systematically explore the molecular mechanisms driving the virus life cycle. Furthermore, we extract and discuss drug repurposing candidates involved in these mechanisms. CoVex renders COVID-19 drug research systems-medicine-ready by giving the scientific community direct access to network medicine algorithms integrating virus-host-drug interactions. It is available at <a href="https://exbio.wzw.tum.de/covex/" target="_blank">this https URL</a>.
  </div>
  

  
  <div class="btn-links">
    








  



















  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-04-14 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-04-14 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/d.b.-blumenthal-l.-viola-and-m.-list-j.-baumbach-p.-tieri-t.-kacprowski/">D.B. Blumenthal, L. Viola, and M. List, J. Baumbach, P. Tieri, <strong>T. Kacprowski</strong></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/blumenthal-et-al.-2020/" itemprop="url">EpiGEN: an epistasis simulation pipeline</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    <strong>Summary</strong> Simulated data is crucial for evaluating epistasis detection tools in genome-wide association studies. Existing simulators are limited, as they do not account for linkage disequilibrium (LD), support limited interaction models of single nucleotide polymorphisms (SNPs) and only dichotomous phenotypes, or depend on proprietary software. In contrast, EpiGEN supports SNP interactions of arbitrary order, produces realistic LD patterns, and can generate both categorical and quantitative phenotypes. <br> <strong>Availability and Impementation</strong> EpiGEN is implemented in Python 3 and is freely available at <a href="https://github.com/baumbachlab/epigen" target="_blank">https://github.com/baumbachlab/epigen</a>. <br> <strong>Supplementary information</strong> A detailed user guide for EpiGEN is available at <em>Bioinformatics</em> online.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa245" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-04-04 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2020-04-04 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-g.-b.-groenning-m.-oubounyt-k.-kanev-j.-b.-lund-t.-kacprowski-d.-zehn-r.-rottger-j.-baumbach/">A. G. B. Groenning, M. Oubounyt, K. Kanev, J. B. Lund,  <strong>T. Kacprowski</strong>, D. Zehn, R. Rottger,  J. Baumbach</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/groenning-et-al.-2020/" itemprop="url">Comparative single-cell trajectory network enrichment identifies pseudo-temporal systems biology patterns in hematopoiesis and CD8 T-cell development</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Single cell transcriptomics (scRNA-seq) technologies allow for investigating cellular processes on an unprecedented resolution. While software packages for scRNA-seq raw data analysis exist, no method for the extraction of systems biology signatures that drive different pseudo-time trajectories exists. Hence, pseudo-temporal molecular sub-network expression profiles remain undetermined, thus, hampering our understanding of the molecular control of cellular development on a single cell resolution. We have developed Scellnetor, the first network-constraint time-series clustering algorithm implemented as interactive webtool to identify modules of genes connected in a molecular interaction network that show differentiating temporal expression patterns. Scellnetor allows selecting two differentiation courses or two developmental trajectories for comparison on a systems biology level. Scellnetor identifies mechanisms driving hematopoiesis in mouse and mechanistically interpretable subnetworks driving dysfunctional CD8 T-cell development in chronic infections. Scellnetor is the first method to allow for single cell trajectory network enrichment for systems level hypotheses generation, thus lifting scRNA-seq data analysis to a systems biology level. It is available as an interactive online tool at <a href="https://exbio.wzw.tum.de/scellnetor/" target="_blank">https://exbio.wzw.tum.de/scellnetor/</a>.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.04.02.021295" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-03-25 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2020-03-25 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-lazarevah.-van-do-hoan-s.-canzar-k.-yuan-j.-baumbach-p.-tieri-t.-kacprowski-m.-list/">O. Lazareva,H. Van Do, Hoan, S. Canzar, K. Yuan, J. Baumbach, P. Tieri, <strong>T. Kacprowski</strong>, M. List</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/lazareva-et-al.-2020/" itemprop="url">BiCoN: Network-constrained biclustering of patients and omics data</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    <strong>Motivation</strong> Unsupervised learning approaches are frequently employed to identify patient subgroups and biomarkers such as disease-associated genes. Thus, clustering and biclustering are powerful techniques often used with expression data, but are usually not suitable to unravel molecular mechanisms along with patient subgroups. To alleviate this, we developed the network-constrained biclustering approach BiCoN (Biclustering Constrained by Networks) which (i) restricts biclusters to functionally related genes connected in molecular interaction networks and (ii) maximizes the difference in gene expression between two subgroups of patients. <br> <strong>Results</strong> Our analyses of non-small cell lung and breast cancer gene expression data demonstrate that BiCoN clusters patients in agreement with known cancer subtypes while discovering gene subnetworks pointing to functional differences between these subtypes. Furthermore, we show that BiCoN is robust to noise and batch effects and can distinguish between high and low load of tumor-infiltrating leukocytes while identifying subnetworks related to immune cell function. In summary, BiCoN is a powerful new systems medicine tool to stratify patients while elucidating the responsible disease mechanism. <br> <strong>Availability</strong> PyPI package: <a href="https://pypi.org/project/bicon" target="_blank">https://pypi.org/project/biconWeb</a> Web interface: <a href="https://exbio.wzw.tum.de/bicon" target="_blank">https://exbio.wzw.tum.de/bicon</a> <br> <strong>Contact</strong> olga.lazareva{at}tum.de <br> <strong>Supplementary information</strong> Supplementary data are available at <em>Bioinformatics</em> online.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.01.31.926345" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-01-20 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2020-01-20 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/pascal-schlosser/">Pascal Schlosser</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/anna-kottgen/">Anna Köttgen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Genetics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/schlosser-et-al.-2020/" itemprop="url">Genetic Studies of Urinary Metabolites Illuminate Mechanisms of Detoxification and Excretion in Humans</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The kidneys integrate information from continuous systemic processes related to the absorption, distribution, metabolism and excretion (ADME) of metabolites. To identify underlying molecular mechanisms, we performed genome-wide association studies of the urinary concentrations of 1,172 metabolites among 1,627 patients with reduced kidney function. The 240 unique metabolite–locus associations (metabolite quantitative trait loci, mQTLs) that were identified and replicated highlight novel candidate substrates for transport proteins. The identified genes are enriched in ADME-relevant tissues and cell types, and they reveal novel candidates for biotransformation and detoxification reactions. Fine mapping of mQTLs and integration with single-cell gene expression permitted the prioritization of causal genes, functional variants and target cell types. The combination of mQTLs with genetic and health information from 450,000 UK Biobank participants illuminated metabolic mediators, and hence, novel urinary biomarkers of disease risk. This comprehensive resource of genetic targets and their substrates is informative for ADME processes in humans and is relevant to basic science, clinical medicine and pharmaceutical research.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41588-019-0567-8" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-03-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2019-03-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/f.-frostsup1/sup/">F. Frost<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-m.-lerch/">M. M. Lerch</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Gastroenterology</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/frost-et-al.-2019/" itemprop="url">Impaired Exocrine Pancreatic Function Associates With Changes in Intestinal Microbiota Composition and Diversity</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Changes in intestinal microbiome composition are associated with inflammatory, metabolic, and malignant disorders. We studied how exocrine pancreatic function affects intestinal microbiota. We performed 16S ribosomal RNA gene sequencing analysis of stool samples from 1795 volunteers from the population-based Study of Health in Pomerania who had no history of pancreatic disease. We also measured fecal pancreatic elastase by enzyme-linked immunosorbent assay and performed quantitative imaging of secretin-stimulated pancreatic fluid secretion. Associations of exocrine pancreatic function with microbial diversity or individual genera were calculated by permutational analysis of variance or linear regression, respectively. Differences in pancreatic elastase levels associated with significantly (P &lt; .0001) greater changes in microbiota diversity than with participant age, body mass index, sex, smoking, alcohol consumption, or dietary factors. Significant changes in the abundance of 30 taxa, such as an increase in Prevotella (q &lt; .0001) and a decrease of Bacteroides (q &lt; .0001), indicated a shift from a type-1 to a type-2 enterotype. Changes in pancreatic fluid secretion alone were also associated with changes in microbial diversity (P = .0002), although to a lesser degree. In an analysis of fecal samples from 1795 volunteers, pancreatic acinar cell, rather than duct cell, function is presently the single most significant host factor to be associated with changes in intestinal microbiota composition.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1053/j.gastro.2018.10.047" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2019-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/cavin-k.-ward-caviness/">Cavin K. Ward-Caviness</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/alanna-c.-morrison/">Alanna C. Morrison</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PloS One</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/ward-caviness-et-al.-2019/" itemprop="url">Mendelian randomization evaluation of causal effects of fibrinogen on incident coronary heart disease.</a>
  </h3>

  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pone.0216222" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-11-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2018-11-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-ligthart/">S. Ligthart</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-dehghansup2/sup/">A. Dehghan<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/b.-z.-alizadehsup2/sup/">B. Z. Alizadeh<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>November 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Am. J. Hum. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/ligthart-et-al.-2018/" itemprop="url">Genome Analyses of &gt;200,000 Individuals Identify 58 Loci for Chronic Inflammation and Highlight Pathways that Link Inflammation and Complex Disorders</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    C-reactive protein (CRP) is a sensitive biomarker of chronic low-grade inflammation and is associated with multiple complex diseases. The genetic determinants of chronic inflammation remain largely unknown, and the causal role of CRP in several clinical outcomes is debated. We performed two genome-wide association studies (GWASs), on HapMap and 1000 Genomes imputed data, of circulating amounts of CRP by using data from 88 studies comprising 204,402 European individuals. Additionally, we performed in silico functional analyses and Mendelian randomization analyses with several clinical outcomes. The GWAS meta-analyses of CRP revealed 58 distinct genetic loci (p &lt; 5 × 10-8). After adjustment for body mass index in the regression analysis, the associations at all except three loci remained. The lead variants at the distinct loci explained up to 7.0% of the variance in circulating amounts of CRP. We identified 66 gene sets that were organized in two substantially correlated clusters, one mainly composed of immune pathways and the other characterized by metabolic pathways in the liver. Mendelian randomization analyses revealed a causal protective effect of CRP on schizophrenia and a risk-increasing effect on bipolar disorder. Our findings provide further insights into the biology of inflammation and could lead to interventions for treating inflammation and its clinical consequences.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.ajhg.2018.09.009" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-09-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-09-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/f.-bonfiglio/">F. Bonfiglio</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-damato/">M. D&rsquo;Amato</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Neurogastroenterol. Motil.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/bonfiglio-et-al.-2018/" itemprop="url">A GWAS meta-analysis from 5 population-based cohorts implicates ion channel genes in the pathogenesis of irritable bowel syndrome</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Irritable bowel syndrome (IBS) shows genetic predisposition, however, large-scale, powered gene mapping studies are lacking. We sought to exploit existing genetic (genotype) and epidemiological (questionnaire) data from a series of population-based cohorts for IBS genome-wide association studies (GWAS) and their meta-analysis. Based on questionnaire data compatible with Rome III Criteria, we identified a total of 1335 IBS cases and 9768 asymptomatic individuals from 5 independent European genotyped cohorts. Individual GWAS were carried out with sex-adjusted logistic regression under an additive model, followed by meta-analysis using the inverse variance method. Functional annotation of significant results was obtained via a computational pipeline exploiting ontology and interaction networks, and tissue-specific and gene set enrichment analyses. Suggestive GWAS signals (P â‰¤ 5.0 Ã— 10-6 ) were detected for 7 genomic regions, harboring 64 gene candidates to affect IBS risk via functional or expression changes. Functional annotation of this gene set convincingly (best FDR-corrected P = 3.1 Ã— 10-10 ) highlighted regulation of ion channel activity as the most plausible pathway affecting IBS risk. Our results confirm the feasibility of population-based studies for gene-discovery efforts in IBS, identify risk genes and loci to be prioritized in independent follow-ups, and pinpoint ion channels as important players and potential therapeutic targets warranting further investigation.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1111/nmo.13358" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-07-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-07-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-pietzner/">M. Pietzner</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-friedrich/">N. Friedrich</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>July 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Endocrinol.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/pietzner-et-al.-2018/" itemprop="url">Empowering thyroid hormone research in human subjects using OMICs technologies</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    OMICs subsume different physiological layers including the genome, transcriptome, proteome and metabolome. Recent advances in analytical techniques allow for the exhaustive determination of biomolecules in all OMICs levels from less invasive human specimens such as blood and urine. Investigating OMICs in deeply characterized population-based or experimental studies has led to seminal improvement of our understanding of genetic determinants of thyroid function, identified putative thyroid hormone target genes and thyroid hormone-induced shifts in the plasma protein and metabolite content. Consequently, plasma biomolecules have been suggested as surrogates of tissue-specific action of thyroid hormones. This review provides a brief introduction to OMICs in thyroid research with a particular focus on metabolomics studies in humans elucidating the important role of thyroid hormones for whole body metabolism in adults.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1530/JOE-18-0117" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-zimmermannsup1/sup/">N. Zimmermann<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-kniefsup1/sup/">J. Knief<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-brabantsup2/sup/">G. Brabant<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-thornssup2/sup/">C. Thorns<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oncotarget</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/zimmermann-et-al.-2018/" itemprop="url">MicroRNA analysis of gastroenteropancreatic neuroendocrine tumors and metastases</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The incidence of neuroendocrine neoplasias (NEN) continues to increase. Since the primary tumor cannot be diagnosed in some cases of metastatic disease, new biomarkers are clearly needed to find the most probable site of origin. Tissue samples from 79 patients were analyzed and microRNA profiles were generated from a total of 76 primary tumors, 31 lymph node and 14 solid organ metastases. NEN metastases were associated with elevated levels of miR-30a-5p, miR-210, miR-339-3p, miR-345 and miR-660. Three microRNAs showed a strong correlation between proliferation index and metastatic disease in general (miR-150, miR-21 and miR-660). Further, each anatomic location (primary or metastatic) had one or more site-specific microRNAs more highly expressed in these tissues. Comparison between primary tumors and metastases revealed an overlap only in pancreatic (miR-127) and ileal tumors (let-7g, miR-200a and miR-331). This thorough analysis of gastroenteropancreatic neuroendocrine tumors demonstrates site-specific microRNA profiles, correlation with proliferation indices as well as corresponding nodal and distant metastases. Using microRNA profiling might improve NEN diagnostics by linking metastases to a most probable site of origin.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.18632/oncotarget.25357" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-wangsup1/sup/">J. Wang<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-kurilshikovsup1/sup/">A. Kurilshikov<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-zhernakova/">A. Zhernakova</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Microbiome</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/wang-et-al.-2018/" itemprop="url">Meta-analysis of human genome-microbiome association studies: the MiBioGen consortium initiative</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    In recent years, human microbiota, especially gut microbiota, have emerged as an important yet complex trait influencing human metabolism, immunology, and diseases. Many studies are investigating the forces underlying the observed variation, including the human genetic variants that shape human microbiota. Several preliminary genome-wide association studies (GWAS) have been completed, but more are necessary to achieve a fuller picture. Here, we announce the MiBioGen consortium initiative, which has assembled 18 population-level cohorts and some 19,000 participants. Its aim is to generate new knowledge for the rapidly developing field of microbiota research. Each cohort has surveyed the gut microbiome via 16S rRNA sequencing and genotyped their participants with full-genome SNP arrays. We have standardized the analytical pipelines for both the microbiota phenotypes and genotypes, and all the data have been processed using identical approaches. Our analysis of microbiome composition shows that we can reduce the potential artifacts introduced by technical differences in generating microbiota data. We are now in the process of benchmarking the association tests and performing meta-analyses of genome-wide associations. All pipeline and summary statistics results will be shared using public data repositories. We present the largest consortium to date devoted to microbiota-GWAS. We have adapted our analytical pipelines to suit multi-cohort analyses and expect to gain insight into host-microbiota cross-talk at the genome-wide level. And, as an open consortium, we invite more cohorts to join us (by contacting one of the corresponding authors) and to follow the analytical pipeline we have developed.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s40168-018-0479-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/f.-frost/">F. Frost</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-m.-lerch/">M. M. Lerch</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Gut</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/frost-et-al.-2018/" itemprop="url">Functional abdominal pain and discomfort (IBS) is not associated with faecal microbiota composition in the general population</a>
  </h3>

  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1136/gutjnl-2018-316502" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2018-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-kular/">L. Kular</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-jagodic/">M. Jagodic</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat Commun</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/kular-et-al.-2018/" itemprop="url">DNA methylation as a mediator of HLA-DRB1*15:01 and a protective variant in multiple sclerosis</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The human leukocyte antigen (HLA) haplotype DRB1*15:01 is the major risk factor for multiple sclerosis (MS). Here, we find that DRB1*15:01 is hypomethylated and predominantly expressed in monocytes among carriers of DRB1*15:01. A differentially methylated region (DMR) encompassing HLA-DRB1 exon 2 is particularly affected and displays methylation-sensitive regulatory properties in vitro. Causal inference and Mendelian randomization provide evidence that HLA variants mediate risk for MS via changes in the HLA-DRB1 DMR that modify HLA-DRB1 expression. Meta-analysis of 14,259 cases and 171,347 controls confirms that these variants confer risk from DRB1*15:01 and also identifies a protective variant (rs9267649, p &lt; 3.32 × 10-8, odds ratio = 0.86) after conditioning for all MS-associated variants in the region. rs9267649 is associated with increased DNA methylation at the HLA-DRB1 DMR and reduced expression of HLA-DRB1, suggesting a modulation of the DRB1*15:01 effect. Our integrative approach provides insights into the molecular mechanisms of MS susceptibility and suggests putative therapeutic strategies targeting a methylation-mediated regulation of the major risk gene.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-018-04732-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-10-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2017-10-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-zellersup1/sup/">T. Zeller<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-schurmannsup1/sup/">C. Schurmann<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/k.-schrammsup1/sup/">K. Schramm<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-mullersup1/sup/">C. Muller<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-kwonsup1/sup/">S. Kwon<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/x.-guosup2/sup/">X. Guo<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-herdersup2/sup/">C. Herder<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-blankenbergsup2/sup/">S. Blankenberg<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-homuthsup2/sup/">G. Homuth<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>October 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Hypertension</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/zeller-et-al.-2017/" itemprop="url">Transcriptome-Wide Analysis Identifies Novel Associations With Blood Pressure</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Hypertension represents a major cardiovascular risk factor. The pathophysiology of increased blood pressure (BP) is not yet completely understood. Transcriptome profiling offers possibilities to uncover genetics effects on BP. Based on 2 populations including 2549 individuals, a meta-analyses of monocytic transcriptome-wide profiles were performed to identify transcripts associated with BP. Replication was performed in 2 independent studies of whole-blood transcriptome data including 1990 individuals. For identified candidate genes, a direct link between long-term changes in BP and gene expression over time and by treatment with BP-lowering therapy was assessed. The predictive value of protein levels encoded by candidate genes for subsequent cardiovascular disease was investigated. Eight transcripts (CRIP1, MYADM, TIPARP, TSC22D3, CEBPA, F12, LMNA, and TPPP3) were identified jointly accounting for up to 13% (95% confidence interval, 8.7-16.2) of BP variability. Changes in CRIP1, MYADM, TIPARP, LMNA, TSC22D3, CEBPA, and TPPP3 expression associated with BP changes-among these, CRIP1 gene expression was additionally correlated to measures of cardiac hypertrophy. Assessment of circulating CRIP1 (cystein-rich protein 1) levels as biomarkers showed a strong association with increased risk for incident stroke (hazard ratio, 1.06; 95% confidence interval, 1.03-1.09; P=5.0×10-5). Our comprehensive analysis of global gene expression highlights 8 novel transcripts significantly associated with BP, providing a link between gene expression and BP. Translational approaches further established evidence for the potential use of CRIP1 as emerging disease-related biomarker.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1161/HYPERTENSIONAHA.117.09458" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-10-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2017-10-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-hertel/">J. Hertel</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-j.-grabe/">H. J. Grabe</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>October 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Sci Rep</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/hertel-et-al.-2017/" itemprop="url">Evidence for Stress-like Alterations in the HPA-Axis in Women Taking Oral Contraceptives</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Using oral contraceptives has been implicated in the aetiology of stress-related disorders like depression. Here, we followed the hypothesis that oral contraceptives deregulate the HPA-axis by elevating circulating cortisol levels. We report for a sample of 233 pre-menopausal women increased circulating cortisol levels in those using oral contraceptives. For women taking oral contraceptives, we observed alterations in circulating phospholipid levels and elevated triglycerides and found evidence for increased glucocorticoid signalling as the transcript levels of the glucocorticoid-regulated genes DDIT4 and FKBP5 were increased in whole blood. The effects were statistically mediated by cortisol. The associations of oral contraceptives with higher FKBP5 mRNA and altered phospholipid levels were modified by rs1360780, a genetic variance implicated in psychiatric diseases. Accordingly, the methylation pattern of FKBP5 intron 7 was altered in women taking oral contraceptives depending on the rs1360780 genotype. Moreover, oral contraceptives modified the association of circulating cortisol with depressive symptoms, potentially explaining conflicting results in the literature. Finally, women taking oral contraceptives displayed smaller hippocampal volumes than non-using women. In conclusion, the integrative analyses of different types of physiological data provided converging evidence indicating that oral contraceptives may cause effects analogous to chronic psychological stressors regarding the regulation of the HPA axis.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41598-017-13927-7" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-09-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2017-09-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-henkel/">R. Henkel</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-hoehndorf/">R. Hoehndorf</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-knupfer/">C. Knupfer</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/w.-liebermeister/">W. Liebermeister</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/d.-waltemath/">D. Waltemath</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Brief. Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/henkel-et-al.-2016/" itemprop="url">Notions of similarity for systems biology models</a>
  </h3>

  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bib/bbw146" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-02-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2017-02-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-jakob/">P. Jakob</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/u.-landmesser/">U. Landmesser</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>February 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Eur. Heart J.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/jakob-et-al.-2017/" itemprop="url">Profiling and validation of circulating microRNAs for cardiovascular events in patients presenting with ST-segment elevation myocardial infarction</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    MicroRNAs (miRNA) are important non-coding modulators controlling patterns of gene expression. However, profiling and validation of circulating miRNA levels related to adverse cardiovascular outcome has not been performed in patients with an acute coronary syndrome (ACS). In a multicentre, prospective ACS cohort, 1002 out of 2168 patients presented with ST-segment elevation myocardial infarction (STEMI). Sixty-three STEMI patients experienced an adjudicated major cardiovascular event (MACE, defined as cardiac death or recurrent myocardial infarction) within 1 year of follow-up. From a miRNA profiling in a matched derivation case-control cohort, 14 miRNAs were selected for validation. Comparing 63 cases vs. 126 controls, 3 miRNAs were significantly differentially abundant. In patients with MACE, miR-26b-5p levels (P = 0.038) were decreased, whereas miR-320a (P = 0.047) and miR-660-5p (P = 0.01) levels were increased. MiR-26b-5p has been suggested to prevent adverse cardiomyocyte hypertrophy, whereas miR-320a promotes cardiomyocyte death and apoptosis, and miR-660-5p has been related to active platelet production. This suggests that miR-26b-5p, miR-320a, and miR-660-5p may reflect alterations of different pathophysiological pathways involved in clinical outcome after ACS. Consistently, these three miRNAs reliably discriminated cases from controls [area under the receiver-operating characteristic curve (AUC) in age- and sex-adjusted Cox regression for miR-26b-5p = 0.707, miR-660-5p = 0.683, and miR-320a =0.672]. Combination of the three miRNAs further increased AUC to 0.718. Importantly, addition of the three miRNAs to both, the Global Registry of Acute Coronary Events (GRACE) score and a clinical model increased AUC from 0.679 to 0.720 and 0.722 to 0.732, respectively, with a net reclassification improvement of 0.20 in both cases. This is the first study performing profiling and validation of miRNAs that are associated with adverse cardiovascular outcome in patients with STEMI. MiR-26b-5p, miR-320a, and miR-660-5p discriminated for MACE and increased risk prediction when added to the GRACE score and a clinical model. These findings suggest that the release of specific miRNAs into circulation may reflect the activation of molecular pathways that impact on clinical outcome after STEMI.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/eurheartj/ehw563" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-pietzner/">M. Pietzner</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/b.-engelmann/">B. Engelmann</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-homuthsup2/sup/">G. Homuth<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-brabantsup2/sup/">G. Brabant<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Med</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/pietzner-et-al.-2017/" itemprop="url">Plasma proteome and metabolome characterization of an experimental human thyrotoxicosis model</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Determinations of thyrotropin (TSH) and free thyroxine (FT4) represent the gold standard in evaluation of thyroid function. To screen for novel peripheral biomarkers of thyroid function and to characterize FT4-associated physiological signatures in human plasma we used an untargeted OMICS approach in a thyrotoxicosis model. A sample of 16 healthy young men were treated with levothyroxine for 8 weeks and plasma was sampled before the intake was started as well as at two points during treatment and after its completion, respectively. Mass spectrometry-derived metabolite and protein levels were related to FT4 serum concentrations using mixed-effect linear regression models in a robust setting. To compile a molecular signature discriminating between thyrotoxicosis and euthyroidism, a random forest was trained and validated in a two-stage cross-validation procedure. Despite the absence of obvious clinical symptoms, mass spectrometry analyses detected 65 metabolites and 63 proteins exhibiting significant associations with serum FT4. A subset of 15 molecules allowed a robust and good prediction of thyroid hormone function (AUC = 0.86) without prior information on TSH or FT4. Main FT4-associated signatures indicated increased resting energy expenditure, augmented defense against systemic oxidative stress, decreased lipoprotein particle levels, and increased levels of complement system proteins and coagulation factors. Further association findings question the reliability of kidney function assessment under hyperthyroid conditions and suggest a link between hyperthyroidism and cardiovascular diseases via increased dimethylarginine levels. Our results emphasize the power of untargeted OMICs approaches to detect novel pathways of thyroid hormone action. Furthermore, beyond TSH and FT4, we demonstrated the potential of such analyses to identify new molecular signatures for diagnosis and treatment of thyroid disorders. This study was registered at the German Clinical Trials Register (DRKS) [DRKS00011275] on the 16th of November 2016.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s12916-016-0770-8" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-y.-chu/">A. Y. Chu</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-s.-fox/">C. S. Fox</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/chu-et-al.-2017/" itemprop="url">Multiethnic genome-wide meta-analysis of ectopic fat depots identifies loci associated with adipocyte development and differentiation</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Variation in body fat distribution contributes to the metabolic sequelae of obesity. The genetic determinants of body fat distribution are poorly understood. The goal of this study was to gain new insights into the underlying genetics of body fat distribution by conducting sample-size-weighted fixed-effects genome-wide association meta-analyses in up to 9,594 women and 8,738 men of European, African, Hispanic and Chinese ancestry, with and without sex stratification, for six traits associated with ectopic fat (hereinafter referred to as ectopic-fat traits). In total, we identified seven new loci associated with ectopic-fat traits (ATXN1, UBE2E2, EBF1, RREB1, GSDMB, GRAMD3 and ENSA; P &lt; 5 × 10-8; false discovery rate &lt; 1%). Functional analysis of these genes showed that loss of function of either Atxn1 or Ube2e2 in primary mouse adipose progenitor cells impaired adipocyte differentiation, suggesting physiological roles for ATXN1 and UBE2E2 in adipogenesis. Future studies are necessary to further explore the mechanisms by which these genes affect adipocyte biology and how their perturbations contribute to systemic metabolic disease.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/ng.3738" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2017-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/e.-kasbohm/">E. Kasbohm</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/b.-holtfreter/">B. Holtfreter</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/u.-volker/">U. Volker</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-petersmann/">A. Petersmann</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-samietz/">S. Samietz</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-biffar/">R. Biffar</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-volzke/">H. Volzke</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-meisel/">P. Meisel</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-homuth/">G. Homuth</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kocher/">T. Kocher</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-teumer/">A. Teumer</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Dent. Res.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/kasbohm-et-al.-2016/" itemprop="url">Exome Variant Analysis of Chronic Periodontitis in 2 Large Cohort Studies</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Periodontitis is characterized by inflammation of the gingival tissue. The main risk factors are socioeconomic factors, sex, age, smoking, and diabetes, but periodontal disease has also a genetic background. Previous genome-wide association studies failed to reveal genome-wide significant associations of single common single-nucleotide polymorphisms with chronic periodontitis. Using the Illumina ExomeChip data of 6,576 participants of the German population-based cohort studies Study of Health in Pomerania (SHIP) and SHIP-Trend, the authors performed single variant and also gene-based association studies of rare and common exonic variations on different periodontal case definitions. Although our study comprised the largest sample size to date to assess genetic predisposition for chronic periodontitis, the authors found no significant association. This study emphasizes that for chronic periodontitis, large sample sizes will be necessary to find genetic associations, even when examining rare genetic variants.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1177/0022034516665076" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-12-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2016-12-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-c.-cornelis/">M. C. Cornelis</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/e.-ingelsson/">E. Ingelsson</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>December 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Hum. Mol. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/cornelis-et-al.-2016/" itemprop="url">Genome-wide association study of caffeine metabolites provides new insights to caffeine metabolism and dietary caffeine-consumption behavior</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Caffeine is the most widely consumed psychoactive substance in the world and presents with wide interindividual variation in metabolism. This variation may modify potential adverse or beneficial effects of caffeine on health. We conducted a genome-wide association study (GWAS) of plasma caffeine, paraxanthine, theophylline, theobromine and paraxanthine/caffeine ratio among up to 9,876 individuals of European ancestry from six population-based studies. A single SNP at 6p23 (near CD83) and several SNPs at 7p21 (near AHR), 15q24 (near CYP1A2) and 19q13.2 (near CYP2A6) met GW-significance (P &lt; 5 × 10-8) and were associated with one or more metabolites. Variants at 7p21 and 15q24 associated with higher plasma caffeine and lower plasma paraxanthine/caffeine (slow caffeine metabolism) were previously associated with lower coffee and caffeine consumption behavior in GWAS. Variants at 19q13.2 associated with higher plasma paraxanthine/caffeine (slow paraxanthine metabolism) were also associated with lower coffee consumption in the UK Biobank (n = 94 343, P &lt; 1.0 × 10-6). Variants at 2p24 (in GCKR), 4q22 (in ABCG2) and 7q11.23 (near POR) that were previously associated with coffee consumption in GWAS were nominally associated with plasma caffeine or its metabolites. Taken together, we have identified genetic factors contributing to variation in caffeine metabolism and confirm an important modulating role of systemic caffeine levels in dietary caffeine consumption behavior. Moreover, candidate genes identified encode proteins with important clinical functions that extend beyond caffeine metabolism.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/hmg/ddw334" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-11-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2016-11-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-lin/">H. Lin</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-t.-ellinor/">P. T. Ellinor</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>November 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Sci Rep</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/lin-et-al.-2016/" itemprop="url">Gene-gene Interaction Analyses for Atrial Fibrillation</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Atrial fibrillation (AF) is a heritable disease that affects more than thirty million individuals worldwide. Extensive efforts have been devoted to the study of genetic determinants of AF. The objective of our study is to examine the effect of gene-gene interaction on AF susceptibility. We performed a large-scale association analysis of gene-gene interactions with AF in 8,173 AF cases, and 65,237 AF-free referents collected from 15 studies for discovery. We examined putative interactions between genome-wide SNPs and 17 known AF-related SNPs. The top interactions were then tested for association in an independent cohort for replication, which included more than 2,363 AF cases and 114,746 AF-free referents. One interaction, between rs7164883 at the HCN4 locus and rs4980345 at the SLC28A1 locus, was found to be significantly associated with AF in the discovery cohorts (interaction OR = 1.44, 95% CI: 1.27-1.65, P = 4.3 × 10-8). Eight additional gene-gene interactions were also marginally significant (P &lt; 5 × 10-7). However, none of the top interactions were replicated. In summary, we did not find significant interactions that were associated with AF susceptibility. Future increases in sample size and denser genotyping might facilitate the identification of gene-gene interactions associated with AF.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/srep35371" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-08-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-08-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-weissmann/">R. Weissmann</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-peper/">M. Peper</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-esche/">J. Esche</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-r.-jensen/">L. R. Jensen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-van-diepen/">L. van Diepen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-port/">M. Port</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-w.-kuss/">A. W. Kuss</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-scherthan/">H. Scherthan</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Health Phys</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/weissmann-et-al.-2016/" itemprop="url">Transcriptome Alterations In X-Irradiated Human Gingiva Fibroblasts</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Ionizing radiation is known to induce genomic lesions, such as DNA double strand breaks, whose repair can lead to mutations that can modulate cellular and organismal fate. Soon after radiation exposure, cells induce transcriptional changes and alterations of cell cycle programs to respond to the received DNA damage. Radiation-induced mutations occur through misrepair in a stochastic manner and increase the risk of developing cancers years after the incident, especially after high dose radiation exposures. Here, the authors analyzed the transcriptomic response of primary human gingival fibroblasts exposed to increasing doses of acute high dose-rate x rays. In the dataset obtained after 0.5 and 5 Gy x-ray exposures and two different repair intervals (0.5 h and 16 h), the authors discovered several radiation-induced fusion transcripts in conjunction with dose-dependent gene expression changes involving a total of 3,383 genes. Principal component analysis of repeated experiments revealed that the duration of the post-exposure repair intervals had a stronger impact than irradiation dose. Subsequent overrepresentation analyses showed a number of KEGG gene sets and WikiPathways, including pathways known to relate to radioresistance in fibroblasts (Wnt, integrin signaling). Moreover, a significant radiation-induced modulation of microRNA targets was detected. The data sets on IR-induced transcriptomic alterations in primary gingival fibroblasts will facilitate genomic comparisons in various genotoxic exposure scenarios.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1097/HP.0000000000000419" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-d.-eichersup1/sup/">J. D. Eicher<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-chamisup1/sup/">N. Chami<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-nomurasup1/sup/">A. Nomura<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-faradaysup2/sup/">N. Faraday<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-l.-auersup2/sup/">P. L. Auer<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-p.-reinersup2/sup/">A. P. Reiner<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-lettresup2/sup/">G. Lettre<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-d.-johnsonsup2/sup/">A. D. Johnson<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>July 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Am. J. Hum. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/eicher-et-al.-2016/" itemprop="url">Platelet-Related Variants Identified by Exomechip Meta-analysis in 157,293 Individuals</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Platelet production, maintenance, and clearance are tightly controlled processes indicative of platelets&rsquo; important roles in hemostasis and thrombosis. Platelets are common targets for primary and secondary prevention of several conditions. They are monitored clinically by complete blood counts, specifically with measurements of platelet count (PLT) and mean platelet volume (MPV). Identifying genetic effects on PLT and MPV can provide mechanistic insights into platelet biology and their role in disease. Therefore, we formed the Blood Cell Consortium (BCX) to perform a large-scale meta-analysis of Exomechip association results for PLT and MPV in 157,293 and 57,617 individuals, respectively. Using the low-frequency/rare coding variant-enriched Exomechip genotyping array, we sought to identify genetic variants associated with PLT and MPV. In addition to confirming 47 known PLT and 20 known MPV associations, we identified 32 PLT and 18 MPV associations not previously observed in the literature across the allele frequency spectrum, including rare large effect (FCER1A), low-frequency (IQGAP2, MAP1A, LY75), and common (ZMIZ2, SMG6, PEAR1, ARFGAP3/PACSIN2) variants. Several variants associated with PLT/MPV (PEAR1, MRVI1, PTGES3) were also associated with platelet reactivity. In concurrent BCX analyses, there was overlap of platelet-associated variants with red (MAP1A, TMPRSS6, ZMIZ2) and white (PEAR1, ZMIZ2, LY75) blood cell traits, suggesting common regulatory pathways with shared genetic architecture among these hematopoietic lineages. Our large-scale Exomechip analyses identified previously undocumented associations with platelet traits and further indicate that several complex quantitative hematological, lipid, and cardiovascular traits share genetic factors.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.ajhg.2016.05.005" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-m.-tajuddinsup1/sup/">S. M. Tajuddin<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/u.-m.-schicksup1/sup/">U. M. Schick<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-lettresup2/sup/">G. Lettre<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-p.-reinersup2/sup/">A. P. Reiner<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-l.-auersup2/sup/">P. L. Auer<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>July 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Am. J. Hum. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/tajuddin-et-al.-2016/" itemprop="url">Large-Scale Exome-wide Association Analysis Identifies Loci for White Blood Cell Traits and Pleiotropy with Immune-Mediated Diseases</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    White blood cells play diverse roles in innate and adaptive immunity. Genetic association analyses of phenotypic variation in circulating white blood cell (WBC) counts from large samples of otherwise healthy individuals can provide insights into genes and biologic pathways involved in production, differentiation, or clearance of particular WBC lineages (myeloid, lymphoid) and also potentially inform the genetic basis of autoimmune, allergic, and blood diseases. We performed an exome array-based meta-analysis of total WBC and subtype counts (neutrophils, monocytes, lymphocytes, basophils, and eosinophils) in a multi-ancestry discovery and replication sample of âˆ¼157,622 individuals from 25 studies. We identified 16 common variants (8 of which were coding variants) associated with one or more WBC traits, the majority of which are pleiotropically associated with autoimmune diseases. Based on functional annotation, these loci included genes encoding surface markers of myeloid, lymphoid, or hematopoietic stem cell differentiation (CD69, CD33, CD87), transcription factors regulating lineage specification during hematopoiesis (ASXL1, IRF8, IKZF1, JMJD1C, ETS2-PSMG1), and molecules involved in neutrophil clearance/apoptosis (C10orf54, LTA), adhesion (TNXB), or centrosome and microtubule structure/function (KIF9, TUBD1). Together with recent reports of somatic ASXL1 mutations among individuals with idiopathic cytopenias or clonal hematopoiesis of undetermined significance, the identification of a common regulatory 3&rsquo; UTR variant of ASXL1 suggests that both germline and somatic ASXL1 mutations contribute to lower blood counts in otherwise asymptomatic individuals. These association results shed light on genetic mechanisms that regulate circulating WBC counts and suggest a prominent shared genetic architecture with inflammatory and autoimmune diseases.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.ajhg.2016.05.003" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-07-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-chamisup1/sup/">N. Chami<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-h.-chensup1/sup/">M. H. Chen<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-j.-slatersup1/sup/">A. J. Slater<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-k.-ganeshsup2/sup/">S. K. Ganesh<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-l.-auersup2/sup/">P. L. Auer<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-d.-johnsonsup2/sup/">A. D. Johnson<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-p.-reinersup2/sup/">A. P. Reiner<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-lettresup2/sup/">G. Lettre<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>July 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Am. J. Hum. Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/chami-et-al.-2016/" itemprop="url">Exome Genotyping Identifies Pleiotropic Variants Associated with Red Blood Cell Traits</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Red blood cell (RBC) traits are important heritable clinical biomarkers and modifiers of disease severity. To identify coding genetic variants associated with these traits, we conducted meta-analyses of seven RBC phenotypes in 130,273 multi-ethnic individuals from studies genotyped on an exome array. After conditional analyses and replication in 27,480 independent individuals, we identified 16 new RBC variants. We found low-frequency missense variants in MAP1A (rs55707100, minor allele frequency [MAF] = 3.3%, p = 2 × 10(-10) for hemoglobin [HGB]) and HNF4A (rs1800961, MAF = 2.4%, p &lt; 3 × 10(-8) for hematocrit [HCT] and HGB). In African Americans, we identified a nonsense variant in CD36 associated with higher RBC distribution width (rs3211938, MAF = 8.7%, p = 7 × 10(-11)) and showed that it is associated with lower CD36 expression and strong allelic imbalance in ex vivo differentiated human erythroblasts. We also identified a rare missense variant in ALAS2 (rs201062903, MAF = 0.2%) associated with lower mean corpuscular volume and mean corpuscular hemoglobin (p &lt; 8 × 10(-9)). Mendelian mutations in ALAS2 are a cause of sideroblastic anemia and erythropoietic protoporphyria. Gene-based testing highlighted three rare missense variants in PKLR, a gene mutated in Mendelian non-spherocytic hemolytic anemia, associated with HGB and HCT (SKAT p &lt; 8 × 10(-7)). These rare, low-frequency, and common RBC variants showed pleiotropy, being also associated with platelet, white blood cell, and lipid traits. Our association results and functional annotation suggest the involvement of new genes in human erythropoiesis. We also confirm that rare and low-frequency variants play a role in the architecture of complex human traits, although their phenotypic effect is generally smaller than originally anticipated.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.ajhg.2016.05.007" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-f.-andlauersup1/sup/">T. F. Andlauer<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/d.-bucksup1/sup/">D. Buck<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/b.-muller-myhsok/">B. Muller-Myhsok</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Sci Adv</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/andlauer-et-al.-2016/" itemprop="url">Novel multiple sclerosis susceptibility loci implicated in epigenetic regulation</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    We conducted a genome-wide association study (GWAS) on multiple sclerosis (MS) susceptibility in German cohorts with 4888 cases and 10,395 controls. In addition to associations within the major histocompatibility complex (MHC) region, 15 non-MHC loci reached genome-wide significance. Four of these loci are novel MS susceptibility loci. They map to the genes L3MBTL3, MAZ, ERG, and SHMT1. The lead variant at SHMT1 was replicated in an independent Sardinian cohort. Products of the genes L3MBTL3, MAZ, and ERG play important roles in immune cell regulation. SHMT1 encodes a serine hydroxymethyltransferase catalyzing the transfer of a carbon unit to the folate cycle. This reaction is required for regulation of methylation homeostasis, which is important for establishment and maintenance of epigenetic signatures. Our GWAS approach in a defined population with limited genetic substructure detected associations not found in larger, more heterogeneous cohorts, thus providing new clues regarding MS pathogenesis.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1126/sciadv.1501678" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-04-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2016-04-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-m.-hillsup1/sup/">S. M. Hill<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-m.-heisersup1/sup/">L. M. Heiser<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-mukherjee/">S. Mukherjee</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nat. Methods</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/hill-et-al.-2016/" itemprop="url">Inferring causal molecular networks: empirical assessment through a community-based effort</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    It remains unclear whether causal, rather than merely correlational, relationships in molecular networks can be inferred in complex biological settings. Here we describe the HPN-DREAM network inference challenge, which focused on learning causal influences in signaling networks. We used phosphoprotein data from cancer cell lines as well as in silico data from a nonlinear dynamical model. Using the phosphoprotein data, we scored more than 2,000 networks submitted by challenge participants. The networks spanned 32 biological contexts and were scored in terms of causal validity with respect to unseen interventional data. A number of approaches were effective, and incorporating known biology was generally advantageous. Additional sub-challenges considered time-course prediction and visualization. Our results suggest that learning causal relationships may be feasible in complex settings such as disease states. Furthermore, our scoring approach provides a practical way to empirically assess inferred molecular networks in a causal sense.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nmeth.3773" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2016-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-c.-pillingsup1/sup/">L. C. Pilling<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-joehanessup1/sup/">R. Joehanes<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-peterssup1/sup/">M. Peters<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-jansensup1/sup/">R. Jansen<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><em>et al.</em></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/b.-penninxsup2/sup/">B. Penninx<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-b.-van-meurssup2/sup/">J. B. van Meurs<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-ferruccisup2/sup/">L. Ferrucci<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kochersup2/sup/">T. Kocher<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-murabitosup2/sup/">J. Murabito<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/d.-melzersup2/sup/">D. Melzer<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Physiol. Genomics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/pilling-et-al.-2016/" itemprop="url">Gene transcripts associated with muscle strength: a CHARGE meta-analysis of 7,781 persons</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Lower muscle strength in midlife predicts disability and mortality in later life. Blood-borne factors, including growth differentiation factor 11 (GDF11), have been linked to muscle regeneration in animal models. We aimed to identify gene transcripts associated with muscle strength in adults. Meta-analysis of whole blood gene expression (overall 17,534 unique genes measured by microarray) and hand-grip strength in four independent cohorts (n = 7,781, ages: 20-104 yr, weighted mean = 56), adjusted for age, sex, height, weight, and leukocyte subtypes. Separate analyses were performed in subsets (older/younger than 60, men/women). Expression levels of 221 genes were associated with strength after adjustment for cofactors and for multiple statistical testing, including ALAS2 (rate-limiting enzyme in heme synthesis), PRF1 (perforin, a cytotoxic protein associated with inflammation), IGF1R, and IGF2BP2 (both insulin like growth factor related). We identified statistical enrichment for hemoglobin biosynthesis, innate immune activation, and the stress response. Ten genes were associated only in younger individuals, four in men only and one in women only. For example, PIK3R2 (a negative regulator of PI3K/AKT growth pathway) was negatively associated with muscle strength in younger (&lt;60 yr) individuals but not older (â‰¥ 60 yr). We also show that 115 genes (52%) have not previously been linked to muscle in NCBI PubMed abstracts. This first large-scale transcriptome study of muscle strength in human adults confirmed associations with known pathways and provides new evidence for over half of the genes identified. There may be age- and sex-specific gene expression signatures in blood for muscle strength.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1152/physiolgenomics.00054.2015" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-10-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2015-10-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-amelingsup1/sup/">S. Ameling<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-k.-chilukotisup1/sup/">R. K. Chilukoti<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-malsch/">C. Malsch</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/v.-liebscher/">V. Liebscher</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/k.-suhre/">K. Suhre</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-pietzner/">M. Pietzner</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-friedrich/">N. Friedrich</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/g.-homuth/">G. Homuth</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/e.-hammer/">E. Hammer</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/u.-volker/">U. Volker</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>October 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Med Genomics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/ameling-et-al.-2015/" itemprop="url">Associations of circulating plasma microRNAs with age, body mass index and sex in a population-based study</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Non-cellular blood circulating microRNAs (plasma miRNAs) represent a promising source for the development of prognostic and diagnostic tools owing to their minimally invasive sampling, high stability, and simple quantification by standard techniques such as RT-qPCR. So far, the majority of association studies involving plasma miRNAs were disease-specific case-control analyses. In contrast, in the present study, plasma miRNAs were analysed in a sample of 372 individuals from a population-based cohort study, the Study of Health in Pomerania (SHIP). Quantification of miRNA levels was performed by RT-qPCR using the Exiqon Serum/Plasma Focus microRNA PCR Panel V3.M covering 179 different miRNAs. Of these, 155 were included in our analyses after quality-control. Associations between plasma miRNAs and the phenotypes age, body mass index (BMI), and sex were assessed via a two-step linear regression approach per miRNA. The first step regressed out the technical parameters and the second step determined the remaining associations between the respective plasma miRNA and the phenotypes of interest. After regressing out technical parameters and adjusting for the respective other two phenotypes, 7, 15, and 35 plasma miRNAs were significantly (q &lt; 0.05) associated with age, BMI, and sex, respectively. Additional adjustment for the blood cell parameters identified 12 and 19 miRNAs to be significantly associated with age and BMI, respectively. Most of the BMI-associated miRNAs likely originate from liver. Sex-associated differences in miRNA levels were largely determined by differences in blood cell parameters. Thus, only 7 as compared to originally 35 sex-associated miRNAs displayed sex-specific differences after adjustment for blood cell parameters. These findings emphasize that circulating miRNAs are strongly impacted by age, BMI, and sex. Hence, these parameters should be considered as covariates in association studies based on plasma miRNA levels. The established experimental and computational workflow can now be used in future screening studies to determine associations of plasma miRNAs with defined disease phenotypes.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s12920-015-0136-7" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-09-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2015-09-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-e.-huffman/">J. E. Huffman</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-l.-smith/">N. L. Smith</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Blood</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/huffman-et-al.-2015/" itemprop="url">Rare and low-frequency variants and their association with plasma levels of fibrinogen, FVII, FVIII, and vWF</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Fibrinogen, coagulation factor VII (FVII), and factor VIII (FVIII) and its carrier von Willebrand factor (vWF) play key roles in hemostasis. Previously identified common variants explain only a small fraction of the trait heritabilities, and additional variations may be explained by associations with rarer variants with larger effects. The aim of this study was to identify low-frequency (minor allele frequency [MAF] â‰¥0.01 and &lt;0.05) and rare (MAF &lt;0.01) variants that influence plasma concentrations of these 4 hemostatic factors by meta-analyzing exome chip data from up to 76,000 participants of 4 ancestries. We identified 12 novel associations of low-frequency (n = 2) and rare (n = 10) variants across the fibrinogen, FVII, FVIII, and vWF traits that were independent of previously identified associations. Novel loci were found within previously reported genes and had effect sizes much larger than and independent of previously identified common variants. In addition, associations at KCNT1, HID1, and KATNB1 identified new candidate genes related to hemostasis for follow-up replication and functional genomic analysis. Newly identified low-frequency and rare-variant associations accounted for modest amounts of trait variance and therefore are unlikely to increase predicted trait heritability but provide new information for understanding individual variation in hemostasis pathways.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1182/blood-2015-02-624551" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-09-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2015-09-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-raffler/">J. Raffler</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/k.-suhre/">K. Suhre</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>PLoS Genet.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/raffler-et-al.-2015/" itemprop="url">Genome-Wide Association Study with Targeted and Non-targeted NMR Metabolomics Identifies 15 Novel Loci of Urinary Human Metabolic Individuality</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Genome-wide association studies with metabolic traits (mGWAS) uncovered many genetic variants that influence human metabolism. These genetically influenced metabotypes (GIMs) contribute to our metabolic individuality, our capacity to respond to environmental challenges, and our susceptibility to specific diseases. While metabolic homeostasis in blood is a well investigated topic in large mGWAS with over 150 known loci, metabolic detoxification through urinary excretion has only been addressed by few small mGWAS with only 11 associated loci so far. Here we report the largest mGWAS to date, combining targeted and non-targeted 1H NMR analysis of urine samples from 3,861 participants of the SHIP-0 cohort and 1,691 subjects of the KORA F4 cohort. We identified and replicated 22 loci with significant associations with urinary traits, 15 of which are new (HIBCH, CPS1, AGXT, XYLB, TKT, ETNPPL, SLC6A19, DMGDH, SLC36A2, GLDC, SLC6A13, ACSM3, SLC5A11, PNMT, SLC13A3). Two-thirds of the urinary loci also have a metabolite association in blood. For all but one of the 6 loci where significant associations target the same metabolite in blood and urine, the genetic effects have the same direction in both fluids. In contrast, for the SLC5A11 locus, we found increased levels of myo-inositol in urine whereas mGWAS in blood reported decreased levels for the same genetic variant. This might indicate less effective re-absorption of myo-inositol in the kidneys of carriers. In summary, our study more than doubles the number of known loci that influence urinary phenotypes. It thus allows novel insights into the relationship between blood homeostasis and its regulation through excretion. The newly discovered loci also include variants previously linked to chronic kidney disease (CPS1, SLC6A13), pulmonary hypertension (CPS1), and ischemic stroke (XYLB). By establishing connections from gene to disease via metabolic traits our results provide novel hypotheses about molecular mechanisms involved in the etiology of diseases.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1371/journal.pgen.1005487" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-07-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2015-07-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/p.-k.-joshisup1/sup/">P. K. Joshi<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-eskosup1/sup/">T. Esko<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-polaseksup2/sup/">O. Polasek<sup>2</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/j.-f.-wilsonsup2/sup/">J. F. Wilson<sup>2</sup></a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>July 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/joshi-et-al.-2015/" itemprop="url">Directional dominance on stature and cognition in diverse human populations</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Homozygosity has long been associated with rare, often devastating, Mendelian disorders, and Darwin was one of the first to recognize that inbreeding reduces evolutionary fitness. However, the effect of the more distant parental relatedness that is common in modern human populations is less well understood. Genomic data now allow us to investigate the effects of homozygosity on traits of public health importance by observing contiguous homozygous segments (runs of homozygosity), which are inferred to be homozygous along their complete length. Given the low levels of genome-wide homozygosity prevalent in most human populations, information is required on very large numbers of people to provide sufficient power. Here we use runs of homozygosity to study 16 health-related quantitative traits in 354,224 individuals from 102 cohorts, and find statistically significant associations between summed runs of homozygosity and four complex traits: height, forced expiratory lung volume in one second, general cognitive ability and educational attainment (P &lt; 1 × 10(-300), 2.1 × 10(-6), 2.5 × 10(-10) and 1.8 × 10(-10), respectively). In each case, increased homozygosity was associated with decreased trait value, equivalent to the offspring of first cousins being 1.2 cm shorter and having 10 months&rsquo; less education. Similar effect sizes were found across four continental groups and populations with different degrees of genome-wide homozygosity, providing evidence that homozygosity, rather than confounding, directly contributes to phenotypic variance. Contrary to earlier reports in substantially smaller samples, no evidence was seen of an influence of genome-wide homozygosity on blood pressure and low density lipoprotein cholesterol, or ten other cardio-metabolic traits. Since directional dominance is predicted for traits under directional evolutionary selection, this study provides evidence that increased stature and cognitive function have been positively selected in human evolution, whereas many important risk factors for late-onset complex diseases may not have been.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nature14618" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-03-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2015-03-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-wahlsup1/sup/">S. Wahl<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-vogtsup1/sup/">S. Vogt<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-grallert/">H. Grallert</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Med</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/wahl-et-al.-2015/" itemprop="url">Multi-omic signature of body weight change: results from a population-based cohort study</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Excess body weight is a major risk factor for cardiometabolic diseases. The complex molecular mechanisms of body weight change-induced metabolic perturbations are not fully understood. Specifically, in-depth molecular characterization of long-term body weight change in the general population is lacking. Here, we pursued a multi-omic approach to comprehensively study metabolic consequences of body weight change during a seven-year follow-up in a large prospective study. We used data from the population-based Cooperative Health Research in the Region of Augsburg (KORA) S4/F4 cohort. At follow-up (F4), two-platform serum metabolomics and whole blood gene expression measurements were obtained for 1,631 and 689 participants, respectively. Using weighted correlation network analysis, omics data were clustered into modules of closely connected molecules, followed by the formation of a partial correlation network from the modules. Association of the omics modules with previous annual percentage weight change was then determined using linear models. In addition, we performed pathway enrichment analyses, stability analyses, and assessed the relation of the omics modules with clinical traits. Four metabolite and two gene expression modules were significantly and stably associated with body weight change (P-values ranging from 1.9 × 10(-4) to 1.2 × 10(-24)). The four metabolite modules covered major branches of metabolism, with VLDL, LDL and large HDL subclasses, triglycerides, branched-chain amino acids and markers of energy metabolism among the main representative molecules. One gene expression module suggests a role of weight change in red blood cell development. The other gene expression module largely overlaps with the lipid-leukocyte (LL) module previously reported to interact with serum metabolites, for which we identify additional co-expressed genes. The omics modules were interrelated and showed cross-sectional associations with clinical traits. Moreover, weight gain and weight loss showed largely opposing associations with the omics modules. Long-term weight change in the general population globally associates with serum metabolite concentrations. An integrated metabolomics and transcriptomics approach improved the understanding of molecular mechanisms underlying the association of weight gain with changes in lipid and amino acid metabolism, insulin sensitivity, mitochondrial function as well as blood cell development and function.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s12916-015-0282-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2015">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2015-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-weissmann/">R. Weissmann</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-huttenrauch/">M. Huttenrauch</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/y.-bouter/">Y. Bouter</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-pradier/">L. Pradier</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-a.-bayer/">T. A. Bayer</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-w.-kuss/">A. W. Kuss</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-wirths/">O. Wirths</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J. Alzheimers Dis.</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/weissmann-et-al.-2015/" itemprop="url">Gene Expression Profiling in the APP/PS1KI Mouse Model of Familial Alzheimer&#39;s Disease</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Alzheimer&rsquo;s disease (AD) is a devastating neurodegenerative disorder characterized by early intraneuronal amyloid-Î² (AÎ²) accumulation, extracellular deposition of AÎ² peptides, and intracellular hyperphosphorylated tau aggregates. These lesions cause dendritic and synaptic alterations and induce an inflammatory response in the diseased brain. Although the neuropathological characteristics of AD have been known for decades, the molecular mechanisms causing the disease are still under investigation. Studying gene expression changes in postmortem AD brain tissue can yield new insights into the molecular disease mechanisms. To that end, one can employ transgenic AD mouse models and the next-generation sequencing technology. In this study, a whole-brain transcriptome analysis was carried out using the well-characterized APP/PS1KI mouse model for AD. These mice display a robust phenotype reflected by working memory deficits at 6 months of age, a significant neuron loss in a variety of brain areas including the CA1 region of the hippocampus and a severe amyloid pathology. Based on deep sequencing, differentially expressed genes (DEGs) between 6-month-old WT or PS1KI and APP/PS1KI were identified and verified by qRT-PCR. Compared to WT mice, 250 DEGs were found in APP/PS1KI mice, while 186 DEGs could be found compared to PS1KI control mice. Most of the DEGs were upregulated in APP/PS1KI mice and belong to either inflammation-associated pathways or lysosomal activation, which is likely due to the robust intraneuronal accumulation of AÎ² in this mouse model. Our comprehensive brain transcriptome study further highlights APP/PS1KI mice as a valuable model for AD, covering molecular inflammatory and immune responses.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3233/JAD-150745" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2014-05-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2014-05-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-j.-grabe/">H. J. Grabe</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/et-al./"><strong><em>et al.</em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-k.-kroemer/">H. K. Kroemer</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>May 2014</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>J Transl Med</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/grabe-et-al.-2014/" itemprop="url">Cohort profile: Greifswald approach to individualized medicine (GANI_MED)</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Individualized Medicine aims at providing optimal treatment for an individual patient at a given time based on his specific genetic and molecular characteristics. This requires excellent clinical stratification of patients as well as the availability of genomic data and biomarkers as prerequisites for the development of novel diagnostic tools and therapeutic strategies. The University Medicine Greifswald, Germany, has launched the &ldquo;Greifswald Approach to Individualized Medicine&rdquo; (GANI_MED) project to address major challenges of Individualized Medicine. Herein, we describe the implementation of the scientific and clinical infrastructure that allows future translation of findings relevant to Individualized Medicine into clinical practice. Clinical patient cohorts (N &gt; 5,000) with an emphasis on metabolic and cardiovascular diseases are being established following a standardized protocol for the assessment of medical history, laboratory biomarkers, and the collection of various biosamples for bio-banking purposes. A multi-omics based biomarker assessment including genome-wide genotyping, transcriptome, metabolome, and proteome analyses complements the multi-level approach of GANI_MED. Comparisons with the general background population as characterized by our Study of Health in Pomerania (SHIP) are performed. A central data management structure has been implemented to capture and integrate all relevant clinical data for research purposes. Ethical research projects on informed consent procedures, reporting of incidental findings, and economic evaluations were launched in parallel.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/1479-5876-12-144" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2014-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2014-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/y.-boutersup1/sup/">Y. Bouter<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong><em>T. Kacprowski<sup>1</sup></em></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/r.-weissmann/">R. Weissmann</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/k.-dietrich/">K. Dietrich</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-borgers/">H. Borgers</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-brauss/">A. Brauss</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-sperling/">C. Sperling</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/o.-wirths/">O. Wirths</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-albrecht/">M. Albrecht</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/l.-r.-jensen/">L. R. Jensen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/a.-w.-kuss/">A. W. Kuss</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-a.-bayer/">T. A. Bayer</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2014</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Front Aging Neurosci</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/bouter-et-al.-2014/" itemprop="url">Deciphering the molecular profile of plaques, memory decline and neuron loss in two mouse models for Alzheimer&#39;s disease by deep sequencing</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    One of the central research questions on the etiology of Alzheimer&rsquo;s disease (AD) is the elucidation of the molecular signatures triggered by the amyloid cascade of pathological events. Next-generation sequencing allows the identification of genes involved in disease processes in an unbiased manner. We have combined this technique with the analysis of two AD mouse models: (1) The 5XFAD model develops early plaque formation, intraneuronal AÎ² aggregation, neuron loss, and behavioral deficits. (2) The Tg4-42 model expresses N-truncated AÎ²4-42 and develops neuron loss and behavioral deficits albeit without plaque formation. Our results show that learning and memory deficits in the Morris water maze and fear conditioning tasks in Tg4-42 mice at 12 months of age are similar to the deficits in 5XFAD animals. This suggested that comparative gene expression analysis between the models would allow the dissection of plaque-related and -unrelated disease relevant factors. Using deep sequencing differentially expressed genes (DEGs) were identified and subsequently verified by quantitative PCR. Nineteen DEGs were identified in pre-symptomatic young 5XFAD mice, and none in young Tg4-42 mice. In the aged cohort, 131 DEGs were found in 5XFAD and 56 DEGs in Tg4-42 mice. Many of the DEGs specific to the 5XFAD model belong to neuroinflammatory processes typically associated with plaques. Interestingly, 36 DEGs were identified in both mouse models indicating common disease pathways associated with behavioral deficits and neuron loss.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fnagi.2014.00075" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2013-11-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2013-11-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-foerster/">S. Foerster</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/v.-m.-dhople/">V. M. Dhople</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/e.-hammer/">E. Hammer</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-herzog/">S. Herzog</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/h.-saafan/">H. Saafan</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/s.-bien-moller/">S. Bien-Moller</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-albrecht/">M. Albrecht</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/u.-volker/">U. Volker</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/c.-a.-ritter/">C. A. Ritter</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>November 2013</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Proteomics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/foerster-et-al.-2013/" itemprop="url">Characterization of the EGFR interactome reveals associated protein complex networks and intracellular receptor dynamics</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Growth factor receptor mediated signaling is meanwhile recognized as a complex signaling network, which is initiated by recruiting specific patterns of adaptor proteins to the intracellular domain of epidermal growth factor receptor (EGFR). Approaches to globally identify EGFR-binding proteins are required to elucidate this network. We affinity-purified EGFR with its interacting proteins by coprecipitation from lysates of A431 cells. A total of 183 proteins were repeatedly detected in high-resolution MS measurements. For 15 of these, direct interactions with EGFR were listed in the iRefIndex interaction database, including Grb2, shc-1, SOS1 and 2, STAT 1 and 3, AP2, UBS3B, and ERRFI. The newly developed Cytoscape plugin ModuleGraph allowed retrieving and visualizing 93 well-described protein complexes that contained at least one of the proteins found to interact with EGFR in our experiments. Abundances of 14 proteins were modulated more than twofold upon EGFR activation whereof clathrin-associated adaptor complex AP-2 showed 4.6-fold enrichment. These proteins were further annotated with different cellular compartments. Finally, interactions of AP-2 proteins and the newly discovered interaction of CIP2A could be verified. In conclusion, a powerful technique is presented that allowed identification and quantitative assessment of the EGFR interactome to provide further insight into EGFR signaling.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1002/pmic.201300154" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2013">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2013-06-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2013-06-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-t.-doncheva/">N. T. Doncheva</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-albrecht/">M. Albrecht</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2013</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Bioinformatics</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/kacprowski-et-al.-2012/" itemprop="url">NetworkPrioritizer: a versatile tool for network-based prioritization of candidate disease genes or other molecules</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The prioritization of candidate disease genes is often based on integrated datasets and their network representation with genes as nodes connected by edges for biological relationships. However, the majority of prioritization methods does not allow for a straightforward integration of the user&rsquo;s own input data. Therefore, we developed the Cytoscape plugin NetworkPrioritizer that particularly supports the integrative network-based prioritization of candidate disease genes or other molecules. Our versatile software tool computes a number of important centrality measures to rank nodes based on their relevance for network connectivity and provides different methods to aggregate and compare rankings. NetworkPrioritizer and the online documentation are freely available at <a href="http://www.networkprioritizer.de" target="_blank">http://www.networkprioritizer.de</a>
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btt164" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2012">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2012-01-01 00:00:00 &#43;0100 CET" itemprop="datePublished">
<meta content="2012-01-01 00:00:00 &#43;0100 CET" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/n.-t.-donchevasup1/sup/">N. T. Doncheva<sup>1</sup></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowskisup1/sup/"><strong>T. Kacprowski<sup>1</sup></strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-albrecht/">M. Albrecht</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 2012</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Wiley Interdiscip Rev Syst Biol Med</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/doncheva-et-al.-2012/" itemprop="url">Recent approaches to the prioritization of candidate disease genes</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Many efforts are still devoted to the discovery of genes involved with specific phenotypes, in particular, diseases. High-throughput techniques are thus applied frequently to detect dozens or even hundreds of candidate genes. However, the experimental validation of many candidates is often an expensive and time-consuming task. Therefore, a great variety of computational approaches has been developed to support the identification of the most promising candidates for follow-up studies. The biomedical knowledge already available about the disease of interest and related genes is commonly exploited to find new gene-disease associations and to prioritize candidates. In this review, we highlight recent methodological advances in this research field of candidate gene prioritization. We focus on approaches that use network information and integrate heterogeneous data sources. Furthermore, we discuss current benchmarking procedures for evaluating and comparing different prioritization methods.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1002/wsbm.1177" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2011-08-01 00:00:00 &#43;0200 CEST" itemprop="datePublished">
<meta content="2011-08-01 00:00:00 &#43;0200 CEST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/d.-emig/">D. Emig</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/t.-kacprowski/"><strong>T. Kacprowski</strong></a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/m.-albrecht/">M. Albrecht</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2011</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>EURASIP J Bioinform Syst Biol</em>
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/emig-et-al.-2011/" itemprop="url">Measuring and analyzing tissue specificity of human genes and protein complexes</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Proteins and their interactions are essential for the survival of each human cell. Knowledge of their tissue occurrence is important for understanding biological processes. Therefore, we analyzed microarray and high-throughput RNA-sequencing data to identify tissue-specific and universally expressed genes. Gene expression data were used to investigate the presence of proteins, protein interactions and protein complexes in different tissues. Our comparison shows that the detection of tissue-specific genes and proteins strongly depends on the applied measurement technique. We found that microarrays are less sensitive for low expressed genes than high-throughput sequencing. Functional analyses based on microarray data are thus biased towards high expressed genes. This also means that previous biological findings based on microarrays might have to be re-examined using high-throughput sequencing results.
  </div>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/1687-4153-2011-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>

      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/r.min.js"></script>
        
      

      
      
    

    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.4e51017b38c7ebaadd7e25fc9503f88c.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
